arches/app/datatypes/core/geojson_feature_collection.py

```diff
@@ -21,6 +21,7 @@ class GeojsonFeatureCollectionDataType(BaseDataType):
     def __init__(self, model=None):
         super(GeojsonFeatureCollectionDataType, self).__init__(model=model)
         self.geo_utils = GeoUtils()
+        self.preferred_srid = 4326
```

Comment:
Why is this called preferred when there is no choice?

Reply:
change in latest